tests/unit_tests/sub_sampling_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
```

Comment:
Also here, the name is very misleading.
Rename it to tdetection sub sampling test please.

Reply:
Changed in b6e26294dec0e4bd447938aeb297500abbedc0ae